packages/config/config/devices/0x0346/alarm_keypad_gen1.json

```diff
@@ -19,6 +19,9 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
+	"compat": {
+		"disableStrictEntryControlDataValidation": true
+	},
```

Comment:
Please move the compat options below the `paramInformation` and before `metadata`